Dockerfile

```diff
@@ -34,8 +34,6 @@ ENV BIGCHAINDB_SERVER_BIND 0.0.0.0:9984
 # but maybe our Docker or Docker Compose stuff does?
 # ENV BIGCHAINDB_API_ENDPOINT http://bigchaindb:9984/api/v1
 
-ENTRYPOINT ["bigchaindb", "--dev-start-rethinkdb", "--dev-allow-temp-keypair"]
+ENTRYPOINT ["bigchaindb"]
 
 CMD ["start"]
-
-EXPOSE 8080 9984 28015 29015
```

Comment:
This is just a hint to docker to expose ports for container networking.
The real option required is --publish over here. We were exposing non-required ports before.